mdagent/tools/maketools.py

```diff
@@ -59,6 +60,7 @@ def make_all_tools(
         ComputeRMSF(path_registry=path_instance),
         ContactsTool(path_registry=path_instance),
         DistanceMatrixTool(path_registry=path_instance),
+        HydrogenBondTool(path_registry=path_instance),
```

Comment:
I though this was only a parent class for the other two methods, why is included in the maketools?

Reply:
please clarify your question. This is the parent class for three of my tools. Doesnt all of the class(basetools) go into maketools?  By the way, I am seriously considering revering my code back to three separate classes(basetools) for all three functions. My code has not been able to be 100% since I reclassified three functions (baker, Kasch, wernet ) from classes (basetools) into subclasses with a parent class, in this case, hydrogenbondTool(basetools).